packages/datetime2/src/components/date-range-picker3/_date-range-picker3.scss

```
@@ -59,6 +59,14 @@
     }
   }
 
+
+  /* stylelint-disable max-line-length */
```

nitpick: I usually like to use `stylelint-disable-next-line` whenever possible to avoid the need to re-enable the lint rule further down.

```suggestion
  // stylelint-disable-next-line max-line-length
```